### ISSUE TITLE
chalk - signature resolution ambiguity in varagrs 

### ISSUE THREAD

**Issue by wojta** [Kotlin/dukat#104]
I tried it with simple console coloring library [Chalk](https://github.com/chalk/chalk) and it results in two main problems.
```kotlin
external interface Chalk {
    @nativeInvoke
    operator fun invoke(vararg text: String): String
    @nativeInvoke
    operator fun invoke(text: TemplateStringsArray, vararg placeholders: String): String 
```
`invoke()` function has ambiguous signature and also I don't know where `TemplateStringsArray` is coming from, probably from TypeScript, but I don't know it very much. It's then `Unresolved reference`.

**Comment by junron** [comment]
[`TemplateStringsArray`](https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules_typedoc_node_modules_typescript_lib_lib_es5_d_.templatestringsarray.html) seems to be a Typescript interface.

**Comment by Schahen** [comment]
I'm happy to anoint that this was actually fixed in dukat@0.0.28
